### DISCUSSION TITLE
EFF on Holiday Dec 23rd-January 2nd

### DISCUSSION THREAD

**Discussion by zoracon** [EFForg/https-everywhere#19871] [Announcements]
Limited support and correspondence from the EFF (@zoracon) during this time. Thank you all for your continued contributions in the repo for HTTPS Everywhere through a very difficult year. There will be some changes coming in 2021 that I hope are welcome and will ease the issues around MV3 and our current load.

I personally am impressed and thankful for such dedication and eye for detail from HTTPSE maintainers.

Happy Holidays and I hope 2021 is kinder to you all. If any of you want an EFF hoodie, stickers, or a shirt sent to you. Please email me at alexis@eff.org.

🥇 

@pipboy96 @J0WI @Bisaloo @cschanaj @TotalCaesar659 @ivysrono

**Comment by ivysrono** [comment]
Could you sent to China?

**Reply by pipboy96** [reply]
@ivysrono I think they would not be able to send it with the return address saying "EFF" since the customs may get interested in it, but they may come up with something.

**Reply by zoracon** [reply]
I can ask if we ever have and if possible!

**Reply by zoracon** [reply]
@ivysrono It looks like we can at the very least send stickers in a blank envelope. Any clothing unfortunately would need a return address. Let me know if that interests you.

**Comment by ivysrono** [comment]
Stickers enough @zoracon

**Reply by zoracon** [reply]
Which encrypted communication do you prefer so I can get an address to send them to?

My preferences are: PGP via email and Keybase. Let me know how you want to communicate and if there are different options as well.

**Reply by pipboy96** [reply]
@ivysrono @zoracon May I suggest also using a service allowing to create self-destructing notes (for example https://secrets.usu.edu/) in addition to PGP in order to prevent your address from staying in email history permanently, in the event @zoracon's key gets stolen in the future?
